edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs

```diff
@@ -85,6 +87,14 @@ public static class Constants
 
         public const string AllModulesIdentifier = "all";
 
+        public const string ProxyImageEnvKey = "ProxyImage";
+
+        public const string ProxyConfigPathEnvKey = "ProxyConfigPath";
+
+        public const string ProxyConfigVolumeEnvKey = "ProxyConfigVolume";
+
+        public const string EdgeAgentServiceAccountName = "ServiceAccountName";
```

Comment:
I forget, why do these need to be in "Core"'s constants?

Reply:
ProxyConfigVolumeEnvKey  is exposed in the appconfig.json/environment as a runtime parameter - "EdgeAgentServiceAccountName" might be the same, but it might not be...

Reply:
Actually, it is used as a runtime parameter in the original code.